packages/theme-data/src/colorSchemes/darkBlue/components/flyout.js

```diff
@@ -0,0 +1,5 @@
+export default {
+  "flyout.borderColor": {
+    value: "transparent"
```

Comment:
We could dry this up a little but setting the base theme value to `transparent`, and only set the color to gray within the web light color scheme.